lib/nexthop.c

```diff
@@ -460,6 +497,27 @@ void nexthop_del_labels(struct nexthop *nexthop)
 	nexthop->nh_label_type = ZEBRA_LSP_NONE;
 }
 
+void nexthop_add_seg6local(struct nexthop *nexthop, uint32_t action,
+			   const struct seg6local_context *ctx)
+{
+	struct seg6local_context *nh_ctx;
+
+	if (action == 0)
```

Comment:
should this test use `ACTION_UNSPEC`? I know that's zero, but...

Reply:
Yes.
I just fix this line with ZEBRA_SEG6_LOCAL_ACTION_UNSPEC.
Thanks.

Reply:
I fixed it.